source-mysql/replication.go

```diff
@@ -345,7 +324,7 @@ func (rs *mysqlReplicationStream) run(ctx context.Context) error {
 						}
 						var sourceInfo = &mysqlSourceInfo{
 							SourceCommon: sourceCommon,
-							EventCursor:  fmt.Sprintf("%s:%d:%d", rs.cursor.Name, rs.cursor.Pos, rowIdx/2),
+							EventCursor:  fmt.Sprintf("%s:%d:%d", cursor.Name, cursor.Pos, rowIdx/2),
```

Comment:
The row images of an update event come in pairs: before-image, then after-image. Up at the top of this loop we have an `if rowIdx%2 == 1` check so that we only process each complete update / pair of row images once, so dividing by two here just makes things look slightly nicer.